modin/numpy/arr.py

```diff
@@ -426,23 +424,20 @@ def max(
                     raise ValueError(
                         f"operand was set up as a reduction along axis 0, but the length of the axis is {out.shape[0]} (it has to be 1)"
                     )
-                return (
-                    fix_dtypes_and_determine_return(
-                        result, 1, dtype, out, (where is not False)
+                if where is not False or out is not None:
+                    return fix_dtypes_and_determine_return(
+                        result, 1, dtype, out, where is not False
                     )
-                    if where is not False or out is not None
-                    else array([initial])
-                )
+                else:
+                    return array([initial])
```

Comment:
technically you can do this:
```python
if where is not False or out is not None:
    return fix_dtypes_and_determine_return(...)
return array([initial])
```
i.e. get rid of `else:` line completely... though I think that it may actually read clearer your way now